polaris/polaris/sep24/deposit.py

```diff
@@ -368,6 +371,7 @@ def deposit(account: str, request: Request) -> Response:
         status=Transaction.STATUS.incomplete,
         to_address=account,
         protocol=Transaction.PROTOCOL.sep24,
+        claimable_balance_supported=claimable_balance_supported,
```

Comment:
Ok, so it seems we already have a column that indicates if the transaction should be submitted via CBs, why do we need the new `status` value?

Reply:
Oh I see, its because we shouldn't use CBs if the account has a trust-line. Ideally, we figure out if it has a trust-line and construct/submit the appropriate transaction without saving anything extra to the transaction object.

Reply:
With the new logic the transaction obj could be `claimable_balance_supported` and have a trustline established.
If thats the case then Polaris does the right thing and submits the transaction as a payment op.